db/query_more_test.go

```diff
@@ -0,0 +1,166 @@
+package db
```

Comment:
These were the old non-json mode query tests. I figured they might cover some use cases not covered by the json mode tests, so I left them here and ported them to json mode.

Reply:
Probably some overlap, but cool